playground/theme/components/TestNuxt.vue

```diff
@@ -15,6 +15,11 @@ defineProps({
 </template>
 
 <style scoped lang="ts">
+const test = keyFrames({
```

Comment:
I suggest calling it lowercase `keyframes` to keep it aligned with MDN. This naming convention matches the naming of the corresponding CSS `@keyframes` rule.

```suggestion
const test = keyframes({
```

https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes